anglia-ruskin-erhvervsakademi-aarhus.csl

```diff
@@ -0,0 +1,710 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Anglia Ruskin Erhvervsakademi Aarhus</title>
+    <title-short>Anglia Ruskin EAAA &amp; BAAA</title-short>
+    <id>http://www.zotero.org/styles/anglia-ruskin-erhvervsakademi-aarhus</id>
+    <link href="http://www.zotero.org/styles/anglia-ruskin-erhvervsakademi-aarhus" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-anglia-ruskin-university" rel="template"/>
+    <author>
+      <name>John Grandt Markvard Høeg</name>
+      <email>saphyron@hotmail.com</email>
+    </author>
+    <contributor>
+      <name>John Høeg</name>
+      <email>saphyron@hotmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Mette G. Hansen</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Edit of Anglia Ruskin University Harvard style for use on Erhvervsakademi Aarhus and Businessakademi Aarhus</summary>
+    <updated>2021-09-15T11:53:44+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
```

Comment:
It looks like you copies all of APAs locales here? Was that on purpose? In which languages is this style effectively going to be used?